src/components/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+
+import React from "react";
+import { Document, Page, Text, View, StyleSheet } from "@react-pdf/renderer";
+
+export interface ParcelClients {
+    packing_datetime: string;
+    collection_centre: string;
+    collection_datetime: string;
+    voucher_number: string;
+    full_name?: string;
+    phone_number?: string;
+    address_1?: string;
+    address_2?: string;
+    address_town?: string;
+    address_county?: string;
+    address_postcode?: string;
+    delivery_instructions?: string;
+    index: number;
+    total: number;
+}
+
+const styles = StyleSheet.create({
+    page: {
+        display: "flex",
+        flexDirection: "column",
+        fontSize: "11px",
+    },
+    cardWrapper: { border: "1pt solid black", margin: "5px" },
+    bold: { fontFamily: "Helvetica-Bold" },
+    row: { display: "flex", flexDirection: "row" },
+    col: { flex: 1, margin: "5px" },
```

Comment:
Change to `flex-grow: 1` for clarity

Reply:
flexGrow: 1 property doesn't work. It seems to like flex: 1

Reply:
huh??? I mean fairs but im so confused by that.